packages/email-plugin/src/plugin.ts

```diff
@@ -247,25 +305,35 @@ export class EmailPlugin implements OnApplicationBootstrap, OnApplicationShutdow
         private emailProcessor: EmailProcessor,
         private jobQueueService: JobQueueService,
         private processContext: ProcessContext,
-        @Inject(EMAIL_PLUGIN_OPTIONS) private options: EmailPluginOptions,
-    ) {}
+        @Inject(EMAIL_PLUGIN_OPTIONS) private options: InitializedEmailPluginOptions,
+    ) { }
 
     /**
      * Set the plugin options.
      */
     static init(options: EmailPluginOptions | EmailPluginDevModeOptions): Type<EmailPlugin> {
-        this.options = options;
+        if (options.templateLoader) {
+            Logger.info(`Using custom template loader '${options.templateLoader.constructor.name}'`);
+        } else if (!options.templateLoader && options.templatePath) {
+            // TODO: this else-if can be removed when deprecated templatePath is removed, 
+            //       because we will either have a custom template loader, or the default loader with a default path
+            options.templateLoader = new FileBasedTemplateLoader(options.templatePath);
+        } else {
+            options.templateLoader = new FileBasedTemplateLoader('./vendure/static/email/templates');
```

Comment:
I am guessing you might have used this path as the default because the `templatePath` docs state "In a default Vendure installation,
the templates are installed to `<project root>/vendure/email/templates`."

These docs are wrong :D 

A project created with `@vendure/create` will actually use `path.join(__dirname, '../static/email/templates')`, where `__dirname` will point to the directory of the `vendure-config.ts` file.

I think we should perhaps throw an exception here, "You must either supply a `templatePath` or provide a custom `templateLoader`", otherwise I feel like whatever default we use will be potentially surprising. Better to force the dev to be explicit about this.

Related: in the `@vendure/create` [config template file](https://github.com/vendure-ecommerce/vendure/blob/master/packages/create/templates/vendure-config.hbs#L88) we should now default to adding:

```ts
templateLoader: new FileBasedTemplateLoader(path.join(__dirname, '../static/email/templates'))
```

rather than using the `templatePath` since it is now deprecated.